src/components/modals/authentification/ModalAuthInitView.tsx

```diff
@@ -0,0 +1,123 @@
+
+/**
+ * Modal vue de base avec boutons google, facebook, apple non actifs
+ */
+type Props = {
+  setContenuModal: React.Dispatch<React.SetStateAction<"init" | "connexion" | "inscription" | 'password-forget'>>
+};
+
+const ModalAuthInitView = ({ setContenuModal}: Props ) => {
+
+  return (
+    <>
+      <div className="flex flex-col items-center w-full px-4 pb-6">
+        <h1 className="text-2xl text-center">Bienvenue !</h1>
+
+        <button
+          type="button"
+          role="button"
+          className="h-11 w-full flex justify-center items-center border border-solid border-vintedTextGrisClair rounded mt-6 cursor-not-allowed"
```

Comment:
Cette className est quasiment la même pour les 3 boutons : "h-11 w-full flex justify-center items-center border border-solid border-vintedTextGrisClair rounded cursor-not-allowed" sont utilisés pour les 3 boutons et ça varie qu'avec "mt-6", "mt-2" et "mt-2 mb-6". Si c bien du css en commun entre les 3 boutons, il faut les définir dans une variable. Pour q'un changement soit partagé entre les 3 boutons

Reply:
corrigé